source/common/common/fine_grain_logger.h

```diff
@@ -127,15 +146,15 @@ class FineGrainLogContext {
 
   /**
    * Returns the current log level of `file`. Default log level is used if there is no
-   * match in log_update_info_.
+   * match in verbosity_update_info_.
    */
   spdlog::level::level_enum getLogLevel(absl::string_view file) const
       ABSL_EXCLUSIVE_LOCKS_REQUIRED(fine_grain_log_lock_);
 
   /**
    * Lock for the following global map and update vector (not for the corresponding loggers).
    */
```

Comment:
seems like this needs to be updated since the fine_grain_log_lock also guards the verbosity_default_level.

Reply:
Good catch, done!